client/src/components/SignUpPage.vue

```diff
@@ -26,17 +27,22 @@
 
 <script>
 import Navbar from "./Navbar";
+import {AUTH_REQUEST} from "../store/auth";
```

Comment:
According to netlify logs, this file is missing and breaking the builds

Reply:
Oh wow, an entire directory wasn't pushed to github. Interesting. Another push happened just now.